apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c

```diff
@@ -42,8 +42,6 @@
 #define SAMPLE_FILE "/mnt/sample_file"
```

Comment:
Please change "operatino" to "operation" in the commit message.

Reply:
Corrected, thanks.